sig-docs/README.md

```diff
@@ -17,6 +17,9 @@ The [charter](charter.md) defines the scope and governance of the Docs Special I
 * APAC SIG Meeting: [Wednesdays at 05:30 UTC](https://docs.google.com/document/d/1emuO4nmaQq3K8JZ9-MQeIygtrCPO9kWv7U7RzTaW4F8/edit) (monthly - Wednesday, after the fourth Tuesday, every month). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=05:30&tz=UTC).
   * [Meeting notes and Agenda](https://docs.google.com/document/d/1emuO4nmaQq3K8JZ9-MQeIygtrCPO9kWv7U7RzTaW4F8/edit).
   * [Meeting recordings](https://www.youtube.com/playlist?list=PL69nYSiGNLP3b5hlx0YV7Lo7DtckM84y8).
+* Arabic Team Meeting: [Wednesdays at 17:00 UTC](https://zoom.us/j/98193594774?pwd=VzlweTE3ckxscDRhWVFtRHp2Znlxdz09) (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=17:00&tz=UTC).
```

Comment:
The timezone converter shows 5 PM UTC, while the agenda doc shows 6 PM UTC as the time for the meeting. Please amend either of the two & let us know for further review.

Reply:
Thanks that catch @divya-mohan0209, I've gone ahead and updated the agenda doc to reflect the correct time **17:00 UTC**.